src/main/java/gregtech/integration/jei/utils/render/ItemStackTextRenderer.java

```diff
@@ -56,8 +56,12 @@ public void render(@NotNull Minecraft minecraft, int xPosition, int yPosition, @
             GlStateManager.scale(0.5, 0.5, 1);
             // z hackery to render the text above the item
             GlStateManager.translate(0, 0, 160);
-
-            String s = (this.chanceBase / 100) + "%";
+            String s;
+            if (chanceBase >= 100 || chanceBase == 0) {
+                s = (this.chanceBase / 100) + "%";
+            } else {
+                s = "." + this.chanceBase + "%";
+            }
```

Comment:
Why is this changing?

Also use `"0."` instead of `"."`.

Reply:
Same here.